### PR TITLE
testing/kgamma5: new aport

### PR DESCRIPTION
**Pull request by PureTryOut**

### CODE REVIEW

File: testing/kgamma5/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
+pkgname=kgamma5
+pkgver=5.16.1
+pkgrel=0
+pkgdesc="Adjust your monitor's gamma settings"
+arch="all"
+url="https://www.kde.org/workspaces/plasmadesktop/"
+license="GPL-2.0-or-later"
+depends_dev="qt5-qtbase-dev kconfig-dev kconfigwidgets-dev kdoctools-dev ki18n-dev"
```
Comment (maxice8) [inline testing/kgamma5/APKBUILD]: depends_dev but no $pkgname-dev subpackage

**Comment by PureTryOut** [comment]
You seem to have been a bit too quick @maxice8, I wanted to fix the `depends_dev` thing still :wink:

**Comment by maxice8** [comment]
It was an accidental push i pulled it to build and test, got the abuild warning, notified you, and mistyped pushp instead of pullp.

**Comment by PureTryOut** [comment]
Ah well, np. I have more Plasma packages with the same mistake I see, will make a PR soon to fix them all in one go.

**Pull request closed without merging**